conans/test/unittests/client/generators/visual_studio_multi_test.py

```diff
@@ -86,3 +86,50 @@ def valid_xml_test(self, use_toolset):
                           "Project=\"conanbuildinfo_release_x64_v141.props\"/>", content_multi)
 
             os.unlink('conanbuildinfo_multi.props')
+
+    def addional_dependencies_test(self):
+
+        def validate_additional_dependencies(libname, additional_dep):
+            tempdir = temp_folder()
+            with chdir(tempdir):
+                settings = Settings.loads(default_settings_yml)
+                settings.os = "Windows"
+                settings.compiler = "Visual Studio"
+                settings.compiler.version = "11"
+                settings.compiler.runtime = "MD"
```

Comment:
Move these settings with the others (around line 108), and remove duplicated ones (`compiler.version`)

Reply:
done